### PR TITLE
[SLS, BS] - DY-1543 Added mocks for rounds by depot

### PR DESCRIPTION
**Pull request by sorenlundskyum**

### CODE REVIEW

File: complete/src/main/java/mock/rbook/handlers/RoundsHandler.java [hunk 1]
```diff
@@ -5,51 +5,82 @@
...
         String trimmedUppercasePostcode = postcode.replaceAll("\\s+", "").toUpperCase();
 
-        final Round round = new Round();
+        final Round round;
 
         switch (trimmedUppercasePostcode) {
             case "GU11TT":
-                round.setRoundNumber("038");
-                round.setDepotCode("130");
+                round = ROUND_038;
```
Comment (Eusebiotrigo) [inline complete/src/main/java/mock/rbook/handlers/RoundsHandler.java]: you could just return the Round here and in each one of the cases.

File: complete/src/main/java/mock/rbook/models/DeliveryPattern.java [hunk 2]
```diff
@@ -4,6 +4,14 @@
 
     private String code;
 
+    public DeliveryPattern() {
```
Comment (Eusebiotrigo) [inline complete/src/main/java/mock/rbook/models/DeliveryPattern.java]: No need for this constructor.

File: complete/src/test/java/mock/rbook/controllers/RoundsControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+
+/**
```
Comment (Eusebiotrigo) [inline complete/src/test/java/mock/rbook/controllers/RoundsControllerTest.java]: You can remove the File Header from the template in intellij 😛

File: complete/src/main/java/mock/rbook/handlers/RoundsHandler.java (hunk 1 above)
Comment (bardyr) [inline complete/src/main/java/mock/rbook/handlers/RoundsHandler.java]: Yes, but I like single exits

File: complete/src/main/java/mock/rbook/models/DeliveryPattern.java (hunk 2 above)
Comment (bardyr) [inline complete/src/main/java/mock/rbook/models/DeliveryPattern.java]: We use both constructors and java will only add the empty one with no other constructors are defined

File: complete/src/test/java/mock/rbook/controllers/RoundsControllerTest.java (hunk 3 above)
Comment (bardyr) [inline complete/src/test/java/mock/rbook/controllers/RoundsControllerTest.java]: Yep

**Pull request merged**